howto/clinic.po

```diff
@@ -111,35 +153,48 @@ msgid ""
 "more sophisticated, and it should be able to do many interesting and smart "
 "things with all the information you give it."
 msgstr ""
+"En travaillant avec Argument Clinic, il faut garder à l'esprit que plus vous "
+"lui donnez de détails, meilleur sera son boulot. Argument Clinic est bien "
```

Comment:
je remplacerais "son boulot" (populaire) par "le résultat".

Reply:
Je voulait respecter le registre de l'auteur, qui emploie "the better job it'll be able to do", le "it'll" est vraiment populaire je pense. Ça casse un peu le coté pas hyper fun du tuto...